app/views/guides/community/before_asking.html.erb

```diff
@@ -97,7 +97,7 @@
     <h3>Conclusion</h3>
 
     <p>
-      Going through each of these tips can take quite a bit of time, but as you get more experience many of these will become part of your development process, which will save you time in the end. If you have not found the root of the problem, after working through each of these tips, it is time to ask others. However, asking others for help is not as simple as it may sound, so please read <a href="/how_to_ask" target="_blank">How to Ask a Technical Question</a>.
+      Going through each of these tips can take quite a bit of time, but as you get more experience many of these will become part of your development process, which will save you time in the end. If you have not found the root of the problem, after working through each of these tips, it is time to ask others. However, asking others for help is not as simple as it may sound, so please read <a href="guides/community/how_to_ask" target="_blank">How to Ask a Technical Question</a>.
```

Comment:
```suggestion
      Going through each of these tips can take quite a bit of time, but as you get more experience many of these will become part of your development process, which will save you time in the end. If you have not found the root of the problem, after working through each of these tips, it is time to ask others. However, asking others for help is not as simple as it may sound, so please read <a href="/guides/community/how_to_ask" target="_blank">How to Ask a Technical Question</a>.
```